sdks/python/container/py35/build.gradle

```diff
@@ -51,6 +51,11 @@ task copyLauncherDependencies(type: Copy) {
   }
 }
 
+task copyDependencyLicenses(type: Copy) {
```

Comment:
Can this task be added to buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy in applyDockerNature?